packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -814,9 +861,31 @@ export class PaginatedMessage {
 	/**
 	 * Custom prompt message when a user wants to jump to a certain page number.
 	 * @default "What page would you like to jump to?"
+	 * @remark To overwrite this property change it somewhere in a "setup" file, i.e. where you also call `client.login()` for your bot.
+	 * @example
+	 * ```typescript
+	 * import { PaginatedMessage } from '@sapphire/discord.js-utilities';
+	 *
+	 * PaginatedMessage.promptMessage = 'Please send the number of the page you would like to jump to.';
+	 * ```
 	 */
 	public static promptMessage = 'What page would you like to jump to?';
 
+	/**
+	 * Custom text to show in front of the page index in the embed footer.
+	 * PaginatedMessage will automatically add a space (` `) after the given text. You do not have to add it yourself.
+	 * @default ""
+	 * @remark To overwrite this property change it somewhere in a "setup" file, i.e. where you also call `client.login()` for your bot.
```

Comment:
Instead of duplicating text shouldn't we link one to the other using a `@seealso` directive?

Reply:
Good call